packages/serverless-workflow-language-service/tests/getFileLanguage.test.ts

```diff
@@ -14,17 +14,15 @@
  * limitations under the License.
  */
 
-import { getFileLanguage, FileLanguage } from "@kie-tools/serverless-workflow-language-service/dist/editor";
+import {
+  FileLanguage,
+  getFileLanguage,
+  getFileLanguageOrThrow,
+} from "@kie-tools/serverless-workflow-language-service/dist/api";
 
 describe("getFileLanguage", () => {
   it("Checking file language with not valid inputs", () => {
-    // @ts-ignore
-    expect(getFileLanguage(null)).toBe(null);
-    // @ts-ignore
-    expect(getFileLanguage(undefined)).toBe(null);
-    // @ts-ignore
     expect(getFileLanguage("")).toBe(null);
-    // @ts-ignore
     expect(getFileLanguage(" ")).toBe(null);
```

Comment:
Removed those annotations because these values are not prohibited as arguments :)